library/core/src/main/java/com/google/android/exoplayer2/drm/ExoMediaDrm.java

```diff
@@ -302,6 +303,15 @@ public String getLicenseServerUrl() {
     public int getRequestType() {
       return requestType;
     }
+
+    public KeyRequest withXDrmInfo(String xDrmInfo) {
```

Comment:
Shouldn't we stick to the stand Java method naming conventions? Since this is a method to set the value of `xDrmInfo` shouldn't it rather be:

```suggestion
    public KeyRequest setXDrmInfo(String xDrmInfo) {
```